src/routes/twitter.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const authController = require('../controllers/auth');
+const { passport } = require('../config/passport');
+
+const router = express.Router();
+
+router.get(
+    '/twitter',
+    passport.authenticate('twitter', { scope: ['profile', 'email', 'openid'] })
+);
+
+router.get(
+    '/twitter/callback',
+    passport.authenticate('twitter', {
+        failureRedirect: '/api/auth/twitter',
```

Comment:
shouldn't this be `/api/twitter-auth/twitter`?

Reply:
yah haha! it's funny that I changed it in google routes but in not in my routes! we have big eyes searching for others' mistakes not ours hahaha!!